lib/website-sales-founder-meeting.ts

```
@@ -417,6 +508,349 @@ function meetingFromAppointment(
+        updated_at: new Date(deps.now()).toISOString(),
+      })
+      .eq("tenant_id", input.tenantId)
+      .eq("lead_id", input.leadId)
+      .eq("id", input.appointmentId)
+      .eq("revision", Number(appointment.revision))
+      .eq("notification_lease_token", notificationLeaseToken)
+      .eq("notification_lease_expires_at", notificationLeaseExpiresAt)
+      .select("id")
+      .maybeSingle();
+    if (released.error || !released.data) {
+      throw new Error(`meeting_notification_lease_release_failed:${released.error?.message || "row_changed"}`);
```

Biome (2.5.8)

[error] Unsafe usage of 'throw'.

(lint/correctness/noUnsafeFinally)